testing/src/cypress/e2e/04-api-institution/03-instition-manage-designation.cy.ts

```diff
@@ -0,0 +1,115 @@
+import InstitutionHelperActions from "../../custom-command/institution/common-helper-functions.cy";
+import Authorization, {
+  ClientId,
+} from "../../custom-command/common/authorization";
+import { CommonRegex } from "../../custom-command/common/commonRegex";
+import { DesignationAgreementStatus } from "../../custom-command/common/designationAgreementStatus";
+
+const institutionHelperActions = new InstitutionHelperActions();
+
+const USERNAME = institutionHelperActions.getUserNameForApiTest();
+const PASSWORD = institutionHelperActions.getUserPasswordForApiTest();
+const TOKEN_URL = institutionHelperActions.getApiUrlForKeyCloakToken();
+enum DesignationAgreementIndex {
+  FirstAgreement = 1,
+  SecondAgreement = 2,
+}
+
+describe("[Designation details] ", () => {
+  let token: string;
+
+  before(async () => {
+    const authorizer = new Authorization();
```

Comment:
minor. new Authorization can be declared as authorization.

Reply:
updated